e2e/client/playwright/monitoring.personal-space.spec.ts

```diff
@@ -60,6 +60,8 @@ test('sending an item from personal space', async ({page}) => {
         page.locator(s('article-item=personal space article 1')),
         'Send to',
     );
+    await page.locator(s('interactive-actions-panel', 'open-popover')).click();
```

Comment:
how was this working before? did anything change?

Reply:
before we had only one Desk ('Sports').. now we have plus one (two in total)... so i need to select on which one i want to send item to avoid error in some cases